models/marts/ledger_current_state/claimable_balances_current.yml

```diff
@@ -0,0 +1,134 @@
+        tests:
+          - not_null:
+              config:
+                where: batch_run_date > current_datetime - interval 2 day
+
+      - name: claimants
+        description: '{{ doc("claimants") }}'
+
+      - name: asset_type
+        description: '{{ doc("asset_type") }}'
+        tests:
+          - not_null:
+              config:
+                where: batch_run_date > current_datetime - interval 2 day
+
+      - name: asset_code
+        description: '{{ doc("asset_code") }}'
+        tests:
+          - not_null:
+              config:
+                where: asset_type != 'native'
+                  and batch_run_date > current_datetime - interval 2 day
+
+      - name: asset_issuer
+        description: '{{ doc("asset_issuer") }}'
+        tests:
+          - not_null:
+              config:
+                where: asset_type != 'native'
+                  and batch_run_date > current_datetime - interval 2 day
+
+      - name: asset_amount
+        description: '{{ doc("asset_amount") }}'
+        tests:
+          - not_null:
+              config:
+                where: batch_run_date > current_datetime - interval 2 day
+
+      - name: sponsor
+        description: '{{ doc("sponsor") }}'
+        tests:
+          - not_null:
+              config:
+                where: batch_run_date > current_datetime - interval 2 day
+
+      - name: flags
+        description: '{{ doc("flags_accounts_balances") }}'
+        tests:
+          - not_null:
+              config:
+                where: batch_run_date > current_datetime - interval 2 day
+
+      - name: last_modified_ledger
+        description: '{{ doc("last_modified_ledger") }}'
+        tests:
+          - not_null:
+              config:
+                where: batch_run_date > current_datetime - interval 2 day
+
+      - name: ledger_entry_change
+        description: '{{ doc("ledger_entry_change") }}'
+        tests:
+          - not_null:
+              config:
+                where: batch_run_date > current_datetime - interval 2 day
+          - accepted_values:
+              values: [0, 1, 2]
+              quote: false
+
+      - name: deleted
+        description: '{{ doc("deleted") }}'
+        tests:
+          - not_null:
+              config:
+                where: batch_run_date > current_datetime - interval 2 day
+
+      - name: batch_id
+        description: '{{ doc("batch_id") }}'
+        tests:
+          - not_null:
+              config:
+                where: batch_run_date > current_datetime - interval 2 day
+
+      - name: batch_run_date
+        description: '{{ doc("batch_run_date") }}'
+        tests:
+          - not_null:
+              config:
+                where: batch_run_date > current_datetime - interval 2 day
+
+      - name: closed_at
+        description: '{{ doc("closed_at") }}'
+        tests:
+          - not_null:
+              config:
+                where: closed_at > current_timestamp - interval 2 day
```

Comment:
Relates to https://github.com/stellar/stellar-dbt-public/pull/66/files#r1689817033
The test: where: <field> should be changed to `batch_run_date` to leverage the table's partitioning

Reply:
Addressed in https://github.com/stellar/stellar-dbt-public/pull/66/commits/fad7bb31de3760302cfbb9cd64c2b077e71a8832